src/Lumi/Components2/Banner.purs

```diff
@@ -102,7 +132,19 @@ actionBanner actions f =
         [ lumiElement box
           $ S._column
           $ S._flex
-          $ S.onDesktop (S._row >>> S._align S.Center)
+          $ S.onDesktop (S._row)
+          $ styleModifier_
+              ( fold
+                  [ css
+                      { alignItems: str "flex-end"
+                      }
+                  , desktopQuery $ css
+                      { alignItems: if (not Array.null props.title)
+                          then str "flex-end"
+                          else str "center"
```

Comment:
What does this part do exactly?

Reply:
According to the designs the positioning of the action button changes depending if the banner content has a title or not.

Reply:
Ah got it! Thanks :)

Reply:
Maybe we could add that in as a comment?